ui-tests/src/main/java/properties/Utils.java

```diff
@@ -0,0 +1,14 @@
+package properties;
+
+/**
+ * Created by mastepan on 10/31/17.
+ */
+public class Utils {
+	public static void sleep(long millis) {
+		try {
+			Thread.sleep(millis);
+		} catch (InterruptedException e) {
+			e.printStackTrace();
+		}
+	}
+}
```

Comment:
I would create a new module that would consist of util classes and also test configuration.